grades/api_test.py

```diff
@@ -449,7 +449,7 @@ def test_successful_program_certificate_generation(self):
         )
         CourseRunGradingStatus.objects.create(course_run=self.run_1, status='complete')
         with mute_signals(post_save):
-            MicromastersCourseCertificate.objects.create(final_grade=final_grade)
+            MicromastersCourseCertificate.objects.create(course=self.run_1.course, user=self.user)
```

Comment:
I think it's probably clearer to go back to storing the `final_grade` variable and to pass the kwargs like this: `(course= final_grade.course_run.course, user=final_grade.user)`